SOURCES/etc/xapi.d/plugins/updater.py

```diff
@@ -181,6 +181,33 @@ def check_update(session, args):
 def update(session, args):
     return install_helper(session, args, 'update')
 
+@error_wrapped
+def query_installed(session, args):
+    packages = args.get('packages')
+    if not packages:
+        return '{}'
+
+    import re
+    packages = re.sub(r'\s+', ' ', packages).replace(',', ' ').split(' ')
+    packages = filter(lambda package: package, packages)
```

Comment:
What input is supported (spaces, commas...) should be specified in the readme and tested in the unit tests.